frontend/public/components/_resource-dropdown.scss

```diff
@@ -9,5 +9,8 @@
     a {
       padding: 4px 20px 4px 6px;
     }
+    .co-truncate {
+      display: block;
```

Comment:
Can we just change the element to a div which is already block instead of adding a CSS rule?

I think probably the API group should just appear below the item without the dash. They're too long to show inline.

Reply:
No need to update `.co-truncate` if `show` is added as it suggested in https://github.com/openshift/console/pull/1145#discussion_r251909081.